scripts/qa_crowbarsetup.sh

```diff
@@ -4541,7 +4541,7 @@ function onadmin_prepare_crowbar_upgrade
         safely crowbar_api_request POST $crowbar_api /installer/upgrade/prepare.json
     else
         safely crowbarctl upgrade prepare
-        wait_for 200 5 "grep current_step /var/lib/crowbar/upgrade/progress.yml | grep -v upgrade_prepare" "prepare step to finish"
+        wait_for 300 5 "grep current_step /var/lib/crowbar/upgrade/progress.yml | grep -v upgrade_prepare" "prepare step to finish"
```

Comment:
I checked [a successful `openstack-mkcloud` job](https://ci.suse.de/job/openstack-mkcloud/44305/consoleFull) and late proposals like trove only took around 2 minutes.  So I guess this might be OK, but it's difficult to be sure without links to example jobs which timed out.